tooling/fixtures/fixture_data/messages.json

```diff
@@ -0,0 +1,12 @@
+[
+    "9872ad089e452c7b6e283dfac2a80d58e8d0ff71cc4d5e310a1debdda4a45f02",
```

Comment:
Are these messages treated as scalars that can be direct inputs in the `sign` operation? If so, I have issues deserializing some of these as scalar values. I may be doing something wrong, but would love some clarification how we use these values.

Reply:
They need to be mapped using `MapMessageToScalarAsHash` prior to passing them in Sign etc. Thanks for pointing out this oversight! Will add the scalar values as well as some `hash_to_sclar` fixtures soon.

Reply:
Much appreciated